### PR TITLE
Fix sensitivity for using with touchable components

### PR DESCRIPTION
**Pull request by ifsnow**
I realized I had some problems with my previous PR(#194).
Sometimes, There may be a touch problem with some devices because of sensitivity.

I think this fix can solve this touch problem.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -188,7 +188,7 @@ class ReactNativeModal extends Component {
 
     this.panResponder = PanResponder.create({
       onMoveShouldSetPanResponder: (evt, gestureState) => {
-        return !(gestureState.dx === 0 && gestureState.dy === 0);
+        return Math.abs(gestureState.dx) >= 4 || Math.abs(gestureState.dy) >= 4;
```
[the remark this answers is not in the document]
Comment (ifsnow) [reply src/index.js]: @mmazzarolo **"return true if user is swiping, return false if it's a single tap."**

Previous ranges(dx=0 & dy=0) are too strict, so tap was not working correctly with some devices. I tested it and now this ranges are reasonable.

[the remark this answers is not in the document]
Comment (mmazzarolo) [reply src/index.js]: Yeah I meant in the code 😁

[the remark this answers is not in the document]
Comment (Aryk) [reply src/index.js:191]: > @mmazzarolo **"return true if user is swiping, return false if it's a single tap."**
> 
> Previous ranges(dx=0 & dy=0) are too strict, so tap was not working correctly with some devices. I tested it and now this ranges are reasonable.

Any reason to not provide a threshold value to developers so they can overwrite it and decide what works best for them?

[the remark this answers is not in the document]
Comment (Aryk) [reply src/index.js:191]: *bump*

**Pull request merged**